core/src/main/java/com/aem/demo/core/services/impl/SearchServiceImpl.java

```diff
@@ -29,17 +29,16 @@ public class SearchServiceImpl implements SearchService {
 
 
     @Reference
-    QueryBuilder queryBuilder;
+    private QueryBuilder queryBuilder;
 
     @Reference
-    ResourceResolverFactory resourceResolverFactory;
+    private ResourceResolverFactory resourceResolverFactory;
 
-    public Map<String, String> createTextSearchQuery(String searchText) {
+    private Map<String, String> createTextSearchQuery(String searchText) {
```

Comment:
Great that you changed visibility here, you're right!)
And now please remember `private` members must be placed under `public` members, so now move it to its righteous place